### PR TITLE
Update CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by michi-covalent**
Document that we are planning to merge Cilium CLI code into cilium/cilium repository.

Ref: https://github.com/cilium/design-cfps/pull/9

### CODE REVIEW

**Comment by michi-covalent** [comment]
cc @joestringer @aanm (i can't seem to @ cilium-maintainers team)

**Comment by joestringer** [comment]
@cilium/cilium-maintainers

File: README.md
Comment (joestringer) [inline README.md:1]: Is the intent to change the user experience for this too? For instance, expect users to download from cilium/cilium, release on a different cadence?

**Comment by michi-covalent** [comment]
> Is the intent to change the user experience for this too? For instance, expect users to download from cilium/cilium, release on a different cadence?

the current proposal is to *not* change the user experience, at least in the short term. we'll continue to publish release artifacts in cilium/cilium-cli repo at its own release cadence independent of cilium/cilium release cycle:

https://github.com/cilium/design-cfps/blame/3b5b719b9fac007d2ea683b18aafe4b3cb62e1eb/cilium/CFP-25694-move-cilium-cli-to-cilium-repo.md#L77-L80

**Comment by joestringer** [comment]
OK, the reason I ask is that this is placing the warning right at the top as the first thing that users see. This looks like a warning for users to get ready to change the way they pull the tool.

Developer-focused warnings should probably be lower down in the readme in a section that's clearly about how you contribute.

**Comment by michi-covalent** [comment]
yeah makes sense. maybe i can update https://github.com/cilium/cilium-cli/blob/main/CONTRIBUTING.md instead 💭

**Comment by pchaigno** [comment]
Is it expected that cilium/cli is also an owner for the README? That doesn't make much sense for me.

**Comment by michi-covalent** [comment]
> Is it expected that cilium/cli is also an owner for the README? That doesn't make much sense for me.

cilium/cli team is the default owner of this repo, so yeah i guess it is sort of expected https://github.com/cilium/cilium-cli/blob/main/CODEOWNERS#L17 do you have any particular team on your mind who should own README.md?

**Comment by pchaigno** [comment]
cilium-cli-maintainers? 😁 Or we split cli into cilium-cli and cilium's CLIs.

**Comment by pchaigno** [comment]
Yeah, so we at least can agree that the description doesn't match the current role of fallback for all of cilium-cli.

**Comment by joestringer** [comment]
The reviewer group for CLI is currently documented as having a project-wide scope: https://docs.cilium.io/en/stable/contributing/development/reviewers_committers/review_process/#project-wide . We can of course revisit this and figure out if there is a better set of individuals who could take ownership of different components.

**Review by pchaigno — APPROVED** [review]

**Comment by michi-covalent** [comment]
https://github.com/cilium/cilium-cli/pull/2710 should reduce the noise to the cli team during the cilium-cli release process 🚀 🙏

**Review by joestringer — APPROVED** [review]

**Pull request merged**